### PR TITLE
Upgade pywinrm to fix Windows workloads for AAP 2.5 EE running Python 3.11

### PR DESCRIPTION
**Pull request by zjleblanc**
Related to issue:
#205 

- update apd-ee-25 pywinrm and consolidate os_windows group_vars
- updated ee build docs

Succesfull run using newly built [execution-environment](https://quay.io/repository/zleblanc/apd-ee-25?tab=tags):
Screenshot 2024-12-13 at 11 55 12 AM

### CODE REVIEW

**Comment by zjleblanc** [comment]
Doing an additional round of testing from scratch to confirm consolidation of windows group_vars does not affect any Windows demos.

File: execution_environments/README.md [hunk 1]
```diff
@@ -6,9 +6,14 @@ Currently these execution environment images are created manually using the `bui
 
 ## Building the execution environment images
 
-1. `podman login registry.redhat.io` in order to pull the base EE images
-2. `./build.sh` to build the EE images and add them to your local podman image cache
+1. Download the [openshift-clients rpm](https://access.redhat.com/downloads/content/rhel---9/x86_64/20821/openshift-clients/4.16.0-202408021139.p0.ge8fb3c0.assembly.stream.el9/x86_64/fd431d51/package)
+2. Overwrite existing link to git-lfs by copying downloaded RPM to `<repo-root>/execution-environments/openshift-clients-4.16.0-202408021139.p0.ge8fb3c0.assembly.stream.el9.x86_64.rpm`
```
Comment (jce-redhat) [inline execution_environments/README.md:10]: can we drop these two bullets for now?  i think git-lfs is the right path forward so the documentation should be updated to take that into account.

**Review by jce-redhat — CHANGES_REQUESTED** [review]

File: execution_environments/README.md (hunk 1 above)
Comment (zjleblanc) [reply execution_environments/README.md]: removed

**Comment by zjleblanc** [comment]
Tested changes with Setup Domain workflow which touches several windows Job Templates
Screenshot 2024-12-13 at 4 54 07 PM

**Pull request merged**